pkg/preparer/orchestrate.go

```diff
@@ -147,6 +147,11 @@ func (p *Preparer) handlePods(podChan <-chan pods.Manifest, quit <-chan struct{}
 			if working {
 				pod := pods.NewPod(manifestToLaunch.ID(), pods.PodPath(p.podRoot, manifestToLaunch.ID()))
 
+				// TODO better solution: force the preparer to have a 0s default timeout, prevent KILLs
+				if pod.Id == POD_ID {
```

Comment:
If a preparer pod has other launchables, they can override the rule in their own `restart_timeout` stanzas.